sql/2020/22_HTTP_2/adoption_of_http_2_by_site_and_requests.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# 22_HTTP_2 - Adoption of HTTP/2 by site and requests
+SELECT 
+  client,
+  firstHtml, 
+  JSON_EXTRACT_SCALAR(payload, "$._protocol") AS http_version, 
+  COUNT(*) AS num_requests,
```

Comment:
`COUNT(0)` for consistency with other queries
```suggestion
  COUNT(0) AS num_requests,
```

Reply:
Sounds good. Agree with the change. Changing all count(*) to count(0) in all files